panic: interface conversion: interface is nil, not interceptor.RTPWriter

### Your environment.
  - Version: v3.0.0-beta.15
  - Browser: Chrome

### What did you do?

In a broadcast scenario, it crashed when adding/removing a shared local track to joining/leaving peers.
I tried a mutex around AddTrack / RemoveTrack / WriteSample with no effect.

### What did you expect?

No crash :)

### What happened?

```
panic: interface conversion: interface is nil, not interceptor.RTPWriter
goroutine 17 [running, locked to thread]:
github.com/pion/webrtc/v3.(*interceptorTrackLocalWriter).WriteRTP(0x40003c6c00, 0x40006a1ef0, 0x4000207400, 0x4a4, 0x4a4, 0x4ba, 0x0, 0x0)
pion/interceptor#11/builds/[...]/.go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.15/interceptor_track_local.go:22 +0x64
github.com/pion/webrtc/v3.(*TrackLocalStaticRTP).WriteRTP(0x4000193830, 0x400039a990, 0x0, 0x0)
pion/interceptor#11/builds/[...]/.go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.15/track_local_static.go:120 +0x1bc
github.com/pion/webrtc/v3.(*TrackLocalStaticSample).WriteSample(0x40002630a0, 0x4000700000, 0x1b57d, 0x1b57d, 0x18d7b473, 0xe7791f71f, 0x114f820, 0x1fca055, 0x0, 0x0)
pion/interceptor#11/builds/[...]/.go/pkg/mod/github.com/pion/webrtc/v3@v3.0.0-beta.15/track_local_static.go:236 +0x1b0
main.pipelineProcessBuffer(0x7f30040810, 0x1b57d, 0x750962a73, 0x1fca055, 0x10)
pion/interceptor#11/builds/[...]/go/driver/gstreamer/pipeline.go:97 +0x1a0
main._cgoexpwrap_a89b420727e6_pipelineProcessBuffer(0x7f30040810, 0x7f0001b57d, 0x750962a73, 0x1fca055, 0x32e5dfdd00000010)
#011_cgo_gotypes.go:308 +0x48
```

--- a/interceptor_track_local.go
+++ b/interceptor_track_local.go
@@ -19,11 +19,9 @@ func (i *interceptorTrackLocalWriter) setRTPWriter(writer interceptor.RTPWriter)
 }
 
 func (i *interceptorTrackLocalWriter) WriteRTP(header *rtp.Header, payload []byte) (int, error) {
-	writer := i.rtpWriter.Load().(interceptor.RTPWriter)
-
-	if writer == nil {
-		return 0, nil
+	if writer, ok := i.rtpWriter.Load().(interceptor.RTPWriter); ok && writer != nil {
+		return writer.Write(&rtp.Packet{Header: *header, Payload: payload}, make(interceptor.Attributes))
 	}
 
-	return writer.Write(&rtp.Packet{Header: *header, Payload: payload}, make(interceptor.Attributes))
+	return 0, nil
 }

